GooglePlacesAutocomplete.js

```diff
@@ -802,42 +808,45 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
       pointerEvents='box-none'
     >
       {!props.textInputHide && (
-        <View
```

Comment:
The diff seems larger than it is. I only wrapped the previous `<View>` under a view to wrap <TextInput> and its component AND added `<BottomComponent>` to new parent view.